sqldelight-compiler/dialect/src/main/kotlin/app/cash/sqldelight/dialect/grammar/mixins/BindParameterMixin.kt

```diff
@@ -0,0 +1,23 @@
+package app.cash.sqldelight.dialect.grammar.mixins
+
+import com.alecstrong.sql.psi.core.psi.SqlBindParameter
+import com.alecstrong.sql.psi.core.psi.SqlCompositeElementImpl
+import com.intellij.lang.ASTNode
+
+abstract class BindParameterMixin(node: ASTNode) : SqlCompositeElementImpl(node), SqlBindParameter {
```

Comment:
this feels like a lot to add into a public API for a pretty specific bug, gonna think on if there's a way to fix this without adding to the API

Reply:
I could add a default value for `replaceWith` too to reduce implementations.